core/camel-api/src/main/java/org/apache/camel/spi/PropertiesComponent.java

```diff
@@ -116,6 +117,25 @@
      */
     Properties loadProperties(Predicate<String> filter);
 
+    /**
+     * Loads the properties from the default locations and sources filtering them out according to a predicate, and maps
+     * the key using the key mapper.
+     * </p>
+     *
+     * <pre>
+     * {
+     *     &#64;code
+     *     PropertiesComponent pc = getPropertiesComponent();
+     *     Properties props = pc.loadProperties(key -> key.startsWith("camel.component.seda"), StringHelper.dashToCamelCase);
+     * }
```

Comment:
```suggestion
     *  <code>
     *     PropertiesComponent pc = getPropertiesComponent();
     *     Properties props = pc.loadProperties(key -> key.startsWith("camel.component.seda"), StringHelper.dashToCamelCase);
     * </code>
```
The formatter broke the Javadoc